shared/src/ledger/ibc/mod.rs

```diff
@@ -22,28 +21,28 @@ where
 {
     // the client counter
     let key = client_counter_key();
-    let value = 0_u64.try_to_vec().unwrap();
+    let value = 0_u64.to_be_bytes().to_vec();
```

Comment:
In ibc-go, `u64` like a counter is encoded with big-endian
https://github.com/cosmos/ibc-go/blob/89ffaafb5956a5ea606e1f1bf249c880bea802ed/modules/core/04-channel/keeper/keeper.go#L115

Reply:
let's add it in a comment
```suggestion
    // In ibc-go, u64 like a counter is encoded with big-endian:
    // https://github.com/cosmos/ibc-go/blob/89ffaafb5956a5ea606e1f1bf249c880bea802ed/modules/core/04-channel/keeper/keeper.go#L115
    let value = 0_u64.to_be_bytes().to_vec();
```